paramiko/client.py

```diff
@@ -383,7 +388,7 @@ def connect(
         )
         if self._log_channel is not None:
             t.set_log_channel(self._log_channel)
-        if banner_timeout is not None:
+            if banner_timeout is not None:
```

Comment:
bad indent here

Reply:
thx for the careful review!
Should be fixed now